.vscode/tasks.json

```diff
@@ -19,7 +19,8 @@
             "group": {
                 "kind": "test",
                 "isDefault": true
-            }
+            },
+            "problemMatcher": []
```

Comment:
Did you intent to commit that file ? (it is already on `master`).

Reply:
I saw it was on master - Not really sure whether we want it or not tbh. Do they normally get committed?

Reply:
I think they don't. I think they don't belong here, especially if there is nothing specific to the project, they should be in a plugin (so it can be maintained and shared with every OCaml projects) or a personal configuration (so it can be modified according to preferences).

Reply:
OK, I'll pull the whole file out. Ta!